### ISSUE TITLE
Include note on bug report that users will not get a response

### ISSUE THREAD

**Issue by aapeliv** [Couchers-org/couchers#4977]
It's annoying to get bug reports that we can't act on. It should say "**You will not get a response**. If you need support, please contact support@couchers.org." or similar just above the submit button.

**Comment by jesseallhands** [comment]
Please be sure to add a subject string to the href so we can (at least in some cases) know the user is contacting support via the bug button rather than the help page:

`"mailto:support@couchers.org?subject=Couchers.org bug report"`

**Comment by bakeiro** [comment]
hmmm maybe can we do a follow up of the issues?

**Comment by bakeiro** [comment]
Pending changes, ticket description on progress

**Comment by jesseallhands** [comment]
For issues where we CAN respond, this should be done through chatwoot so we can track the issues and responses

**Comment by bakeiro** [comment]
> For issues where we CAN respond, this should be done through chatwoot so we can track the issues and responses

@jesseallhands 
yeah that's the thing, my idea was to add an input email for the issues that don't have a user name

**Comment by bakeiro** [comment]
@jesseallhands 
if that's the case why adding this link to send an email?


>Please be sure to add a subject string to the href so we can (at least in some cases) know the user is contacting support via the bug button rather than the help page:
"mailto:support@couchers.org?subject=Couchers.org bug report"

**Comment by bakeiro** [comment]
>For issues where we CAN respond, this should be done through chatwoot so we can track the issues and responses

This means for people who has an couchers account? My idea was to put an input e-mail so the users that **don't** have a couchers account are reachable, so we don't lose feedback/information from them...

but I guess it's better to use chatwoot then? [not sure how this tool works though😅]

tell me what do you think, maybe both things makes sense?

**Comment by aapeliv** [comment]
Well the issue is that the bugs come here, which is public. I don’t think people necessarily want their email public?

**Comment by kamilkaczmareksolutions** [comment]
I'd like to take this if it's still relevant and you have review capacity. Happy to open a small PR from a fork once confirmed.

cc @nabramow

**Comment by aapeliv** [comment]
@kamilkaczmareksolutions yes please feel free to take this on, it is still relevant
